src/download.rs

```diff
@@ -173,7 +173,7 @@ impl Imap {
         info!(context, "Downloading message {}/{} fully...", folder, uid);
 
         let (_, error_cnt) = self
-            .fetch_many_msgs(context, folder, vec![uid], false, false)
+            .fetch_many_msgs(context, folder, vec![uid], false, false, &mut Vec::new())
```

Comment:
This `Vec::new()` instead of ignoring the return value with `_` looks really strange for rust code.